.github/workflows/ui-tests.yaml

```diff
@@ -34,10 +36,33 @@ jobs:
         run: |
           sudo echo "127.0.0.1 local.kyma.dev" | sudo tee -a /etc/hosts
           wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | sudo bash
-          IMG=europe-docker.pkg.dev/kyma-project/dev/api-gateway-manager:PR-${{ github.event.number }} ./tests/ui/tests/scripts/k3d-ci-kyma-dashboard-integration.sh
+          IMG=europe-docker.pkg.dev/kyma-project/dev/api-gateway-manager:PR-${{ github.event.number }} ./tests/ui/tests/scripts/k3d-ci-kyma-dashboard-integration.sh prod
       - uses: actions/upload-artifact@v4
         if: always()
         name: Export Cypress output
         with:
           name: cypress
           path: tests/ui/tests/cypress/
+
+  run-ui-tests-periodic:
+    name: Run UI tests
```

Comment:
We should have a different display name for this, since it can be confusing when it's in the same workflow. Also having it in the same workflow will now display it as skipped on every PR. Maybe it's better to add all jobs running on schedule in its own workflow.

Reply:
Right, I've just done that